data/scripts.ts

```diff
@@ -345,6 +345,7 @@ export const Scripts: BattleScriptsData = {
 				this.add('-fail', pokemon);
 				this.attrLastMove('[still]');
 			}
+			if (move.isFutureMove) return true;
 			return false;
```

Comment:
We already have `this.NOT_FAIL` for other moves that don't double the power of Stomping Tantrum. Maybe this should say
```suggestion
			return hitResult === this.NOT_FAIL;
```
(with other related changes as necessary)

Reply:
trySpreadMoveHit returns a boolean. We _could_ refactor it, but I think it's appropriate that it return true, i.e., futuresight _does_ succeed when it is used. Thoughts?

Reply:
I'm not changing the return value of `trySpreadMoveHit` though am I?

Reply:
Gotcha. Resolved.

Reply:
Great, although personally I really don't like the `if (...) return true; return false;` especially when the condition actually has boolean type.

Reply:
updated